rust_snuba/src/processors/metrics_summaries.rs

```diff
@@ -1,25 +1,35 @@
 use crate::config::ProcessorConfig;
 use anyhow::Context;
+use chrono::DateTime;
 use rust_arroyo::backends::kafka::types::KafkaPayload;
 use serde::{Deserialize, Serialize};
+use serde_json::value::RawValue;
 use std::collections::BTreeMap;
 use uuid::Uuid;
 
 use crate::processors::utils::{enforce_retention, hex_to_u64};
-use crate::types::{InsertBatch, KafkaMessageMetadata};
+use crate::types::{InsertBatch, KafkaMessageMetadata, RowData};
 
 pub fn process_message(
     payload: KafkaPayload,
     _: KafkaMessageMetadata,
     config: &ProcessorConfig,
 ) -> anyhow::Result<InsertBatch> {
     let payload_bytes = payload.payload().context("Expected payload")?;
-    let from: InputMessage = serde_json::from_slice(payload_bytes)?;
+    let msg: InputMessage = serde_json::from_slice(payload_bytes)?;
 
+    if msg._metrics_summary.is_none() {
+        return Ok(InsertBatch {
```

Comment:
you can shorten it to `Ok(InsertBatch::default())`, or i think in the generic-metrics PR there's an alias `InsertBatch::skip()` as well